packages/codemirror-lang-dentaku/src/builtInFunctions.ts

```diff
@@ -1,69 +1,77 @@
-export const builtInFunctions = [
+/**
+ * The supported built-in functions names and their config.
+ *
+ * When `null` config is provided, the `minArgs` is assumed to be 0 and `maxArg`
+ * to be Infinity
+ */
+export const builtInFunctions = {
   // Math
-  "acos",
-  "acosh",
-  "asin",
-  "asinh",
-  "atan",
-  "atan2",
-  "atanh",
-  "cbrt",
-  "cos",
-  "cosh",
-  "erf",
-  "erfc",
-  "exp",
-  "frexp",
-  "gamma",
-  "hypot",
-  "ldexp",
-  "lgamma",
-  "log",
-  "log10",
-  "log2",
-  "sin",
-  "sinh",
-  "sqrt",
-  "tan",
-  "tanh",
+  acos: { minArgs: 1, maxArgs: 1 },
```

Comment:
praise: thanks for being so thorough in writing up all those argument counts :)

Reply:
Most were fixed arity already, so it took just a couple of hours to dig into Dentaku's unit tests to find out the others. ;)